src/lib/header-bidding/a9/a9.ts

```diff
@@ -70,21 +65,69 @@ const requestBids = async (
 		return requestQueue;
 	}
 
-	requestQueue = requestQueue
-		.then(
-			() =>
-				new Promise<void>((resolve) => {
-					window.apstag?.fetchBids({ slots: adUnits }, () => {
-						window.googletag.cmd.push(() => {
-							window.apstag?.setDisplayBids();
-							resolve();
-						});
-					});
-				}),
-		)
-		.catch(() => {
-			reportError(new Error('a9 header bidding error'), 'commercial');
-		});
+	// so now we need to look over the `adunits` and check their slotID
+	// so that we can block the bidders that we want to block based on the slotID
+
+	const section = window.guardian.config.page.section;
+	const isFront = window.guardian.config.page.isFront;
+
+	const isNetworkFront =
+		isFront &&
+		['uk', 'us', 'au', 'europe', 'international'].includes(section);
+
+	const isSectionFront =
+		isFront &&
+		['commentisfree', 'sport', 'culture', 'lifeandstyle'].includes(section);
```

Comment:
I can see you are checking all the Network Fronts on L76 but for the Section Fronts you are checking 4 of them while I believe there are much more Section Fronts or is GumGum showing just for those specific ones?